common/event_modifiers/wc_event_modifiers.txt

```diff
@@ -1701,4 +1701,9 @@ writing_whispers = {
 	}
 
 	monthly_character_piety = 0.25
+}
+twilight_architecture = {
+	icon = 75
```

Comment:
I would change the icon to something more associated with building: 4 or 54 icons.
https://ck2.paradoxwikis.com/Modifiers

Reply:
![image](https://user-images.githubusercontent.com/31159937/77860957-6bb19780-7212-11ea-89c9-136c6f4c86ed.png)

Reply:
horns = spikes imo